src/wallet/rpcwallet.cpp

```diff
@@ -1754,7 +1754,7 @@ static RPCHelpMan gettransaction()
 
     if (verbose) {
         UniValue decoded(UniValue::VOBJ);
-        TxToUniv(*wtx.tx, uint256(), decoded, false);
+        TxToUniv(*wtx.tx, /* hashBlock */ uint256(), /* entry */ decoded, /* include_hex */ false);
```

Comment:
If you keep this commit in this pull, it could make sense to use the `/* arg= */` syntax, so that clang-tidy understands it as well.

Reply:
Good suggestion. My plan is to do this and squash it into a0f2dab816fc9987fb177c02f5d801cf0b8ebbf9